HW03/maksim_manzhai/maksim_manzhai_hw_03_t_03.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  def task3(log)
+    lines = log.split(/\n/)
+    lines.select! { |line| event_valid?(line) }.map! { |line| Time.parse(line) }
+
+    return '0' if lines.empty? || lines.size == 1
+
+    duration(lines.compact)
```

Comment:
we can write it in one line
```suggestion
    lines.size <= 1 ? '0' : duration(lines.compact)
```